tests/utils.py

```diff
@@ -1,50 +1,68 @@
-import os
-import sys
-from contextlib import contextmanager
-from pathlib import Path
-
-from click.testing import CliRunner
-
-ROOT_DIR = Path(__file__).parent.parent
-EXAMPLE_AGENT_DIR = ROOT_DIR / "example_agents"
-RANDOM_AGENT_DIR = EXAMPLE_AGENT_DIR / "random"
-CHATBOT_AGENT_DIR = EXAMPLE_AGENT_DIR / "chatbot"
-ACME_DQN_AGENT_DIR = EXAMPLE_AGENT_DIR / "acme_dqn"
-ACME_R2D2_AGENT_DIR = EXAMPLE_AGENT_DIR / "acme_r2d2"
-GH_SB3_AGENT_DIR = EXAMPLE_AGENT_DIR / "gh_sb3_agent"
-RLLIB_AGENT_DIR = EXAMPLE_AGENT_DIR / "rllib_agent"
-PAPAG_AGENT_DIR = EXAMPLE_AGENT_DIR / "papag"
-SB3_AGENT_DIR = EXAMPLE_AGENT_DIR / "sb3_agent"
-TEST_VENV_AGENT_DIR = ROOT_DIR / "tests" / "test_agents" / "venv_agent"
-
-TESTING_GITHUB_ACCOUNT = "agentos-project"
-TESTING_GITHUB_REPO = "agentos"
-TESTING_GITHUB_REPO_URL = (
-    f"https://github.com/{TESTING_GITHUB_ACCOUNT}/{TESTING_GITHUB_REPO}"
-)
-TESTING_BRANCH_NAME = "test_staging"
-
-
-def run_test_command(cmd, cli_args=None, cli_kwargs=None):
-    call_list = [a for a in (cli_args or [])]
-    for param, val in (cli_kwargs or {}).items():
-        call_list.append(param)
-        call_list.append(val)
-    runner = CliRunner()
-    print(f"Running the following: {cmd.name} {call_list}")
-    result = runner.invoke(cmd, call_list, catch_exceptions=False)
-    assert result.exit_code == 0, result.output
-
-
-@contextmanager
-def run_in_dir(dir_path):
-    curr_dir = os.getcwd()
-    os.chdir(dir_path)
-    try:
-        yield
-    finally:
-        os.chdir(curr_dir)
-
-
-def is_linux():
-    return "linux" in sys.platform
+import os
+import sys
+from contextlib import contextmanager
+from pathlib import Path
+
+from click.testing import CliRunner
+
+ROOT_DIR = Path(__file__).parent.parent
+EXAMPLE_AGENT_DIR = ROOT_DIR / "example_agents"
+RANDOM_AGENT_DIR = EXAMPLE_AGENT_DIR / "random"
+CHATBOT_AGENT_DIR = EXAMPLE_AGENT_DIR / "chatbot"
+ACME_DQN_AGENT_DIR = EXAMPLE_AGENT_DIR / "acme_dqn"
+ACME_R2D2_AGENT_DIR = EXAMPLE_AGENT_DIR / "acme_r2d2"
+GH_SB3_AGENT_DIR = EXAMPLE_AGENT_DIR / "gh_sb3_agent"
+RLLIB_AGENT_DIR = EXAMPLE_AGENT_DIR / "rllib_agent"
+PAPAG_AGENT_DIR = EXAMPLE_AGENT_DIR / "papag"
+SB3_AGENT_DIR = EXAMPLE_AGENT_DIR / "sb3_agent"
+TEST_VENV_AGENT_DIR = ROOT_DIR / "tests" / "test_agents" / "venv_agent"
+
+TESTING_GITHUB_ACCOUNT = "agentos-project"
+TESTING_GITHUB_REPO = "agentos"
+TESTING_GITHUB_REPO_URL = (
+    f"https://github.com/{TESTING_GITHUB_ACCOUNT}/{TESTING_GITHUB_REPO}"
+)
+TESTING_BRANCH_NAME = "test_staging"
```

Comment:
It's weird that the Github diff is shows that the first 25 lines of this file were deleted and added but the first 25 lines of the file that were added appear to be identical to the ones that were deleted. I wonder why the diff isn't only showing the updated lines starting at line 28.

Am I missing something (e.g., did white space change)?

Reply:
Yeah, for some reason this file had windows line terminators, so I converted it to unix terminators.  I assume this is causing the weirdness you're seeing!